src/main/resources/org/jenkinsci/plugins/workflow/support/steps/stash/StashStep/help.html

```diff
@@ -1,6 +1,7 @@
 <div>
     Saves a set of files for use later in the same build, generally on another node/workspace.
```

Comment:
I think the fact that stashes are not available across jobs is already covered by the phrasing `same build` here. Maybe we should bold `same build`?

Reply:
AFAIK the Jenkins terminology is:

Jenkins has multiple `jobs`, that have multiple `builds`.
Each `build` can be a `pipeline` that has multiple `stages`.
Each `stage` has multiple `steps`.

So using the term `build` here is misleading.

From a build (pipeline) a job can trigger another job. The statement `on another node/workspace` may mislead users to believe that they can stash between the triggered job and the trigger job.

Reply:
Yes, that is the terminology, `same build` means that the stash can only be used from the same build of the same job. You cannot access a stash from a different build of the same job, or from a build of a different job entirely. I think `on another node/workspace` is mentioned because if you are still in the same node block, then you can just use a shell script to access the files directly, so there is no point of using `stash`. Maybe changing `same build` to `same build of the same job` to be more explicit would help?

Reply:
You are right, the phrasing is clear. But for someone who doesn't work with Jenkins all day every day, a more explicit phrasing could make the difference for hours of frustration. I don't think we need to change this line at all. The bellow completion clarifies if someone had any doubts.

Reply:
@ovidiub13 
It might be worth changing this line to 
```suggestion
    Saves a set of files for use later in the same Pipeline run, generally on another node/workspace.
```

Reply:
For pipelines this is more accurate:
```suggestion
    Saves a set of files for use later on any node/workspace in the same Pipeline run.
```

Reply:
This still can lead to the misconception that a triggered build is part of this pipeline, or maybe that's just me... Anyway, I like this last suggestion the best. Thank you both.